lib/src/value_objects/enums.dart

```diff
@@ -96,3 +96,23 @@ extension SupplierAccountTypeEx on SupplierAccountType {
     }
   }
 }
+
+/// [Assistant] is the user assistant
+enum Assistant {
+  bowi,
+  bev,
+}
+
+/// [AssistantExt] offers an extension to get string names of the [Assistant]
+extension AssistantExt on Assistant {
+  String get name {
+    switch (this) {
+      case Assistant.bev:
+        return 'BEV';
+      case Assistant.bowi:
+        return 'BOWI';
+      default:
```

Comment:
here you can just have one `case` and the other as `default`.
Alternatively use an if statement
`if(this==Assistant.bev) {return 'BEV';} return 'BOWI';`

Reply:
Okay. Fixing it.

Reply:
Fixed